prow/plugins/plugins.go

```diff
@@ -349,30 +379,28 @@ type Size struct {
 	Xxl int `json:"xxl"`
 }
 
-/*
-  Blockade specifies a configuration for a single blockade.blockade. The configuration for the
-  blockade plugin is defined as a list of these structures. Here is an example of a complete
-  yaml config for the blockade plugin that is composed of 2 Blockade structs:
-
-	blockades:
-	- repos:
-	  - kubernetes-incubator
-	  - kubernetes/kubernetes
-	  - kubernetes/test-infra
-	  blockregexps:
-	  - 'docs/.*'
-	  - 'other-docs/.*'
-	  exceptionregexps:
-	  - '.*OWNERS'
-	  explanation: "Files in the 'docs' directory should not be modified except for OWNERS files"
-	- repos:
-	  - kubernetes/test-infra
-	  blockregexps:
-	  - 'mungegithub/.*'
-	  exceptionregexps:
-	  - 'mungegithub/DeprecationWarning.md'
-	  explanation: "Don't work on mungegithub! Work on Prow!"
-*/
+// Blockade specifies a configuration for a single blockade. The configuration for the
+// blockade plugin is defined as a list of these structures. Here is an example of a complete
+// yaml config for the blockade plugin that is composed of 2 Blockade structs:
```

Comment:
I struggled with writing a helpful comment to most of these config structs. I opted to unify them at the end until I have a better idea.

I agree with the fact to having sample configuration here is not a good idea. Should I just shorten them to something like:

```go
// Blockade specifies a configuration for a single blockade. The configuration for the
// blockade plugin is defined as a list of these structures.
```

Reply:
Let's just not add them.

Reply:
Do we provide sample yaml somewhere else? Like in a markdown file? If so then just describing the purpose of this struct sans yaml sounds sufficient

Reply:
We do provide it in the markdown docs